openfisca_us/parameters/gov/states/il/tax/income/exemption/personal.yaml

```diff
@@ -0,0 +1,18 @@
+description: Illinois personal exemption allowance
+ELIGIBLE:
```

Comment:
Make this a single number ($2,375 in 2021) per Section 204 of the [legal code](https://www.ilga.gov/legislation/ilcs/ilcs4.asp?DocName=003500050HArt%2E+2&ActID=577&ChapterID=8&SeqStart=600000&SeqEnd=3100000)